youtube_dl/extractor/npo.py

```diff
@@ -184,21 +186,29 @@ def _real_extract(self, url):
         return self._get_info(url, video_id) or self._get_old_info(video_id)
 
     def _get_info(self, url, video_id):
-        token = self._download_json(
+        _, xsrf_token_response = self._download_webpage_handle(
             'https://www.npostart.nl/api/token', video_id,
             'Downloading token', headers={
                 'Referer': url,
                 'X-Requested-With': 'XMLHttpRequest',
-            })['token']
+            })
```

Comment:
Avoid reading the page content:
```suggestion
        xsrf_token_response = self._request_webpage(
            'https://www.npostart.nl/api/token', video_id,
            'Downloading token', headers={
                'Referer': url,
                'X-Requested-With': 'XMLHttpRequest',
            })
```

Reply:
I closed some of these because XSRF is no longer necessary fortunately. The code this comments on is no longer is in the diff.